device/cuda/src/clusterization/clusterization_algorithm.cu

```diff
@@ -72,13 +76,17 @@ __device__ void fast_sv_1(index_t* f, index_t* gf, unsigned char adjc,
          * cluster ID if it is lower than ours, essentially merging the two
          * together.
          */
-        __builtin_assume(adjc <= 8);
-        for (unsigned char k = 0; k < adjc; ++k) {
-            index_t q = gf[adjv[k]];
+        for (index_t tst = 0; tst < CELLS_PER_THREAD; ++tst) {
+            const index_t cid = tst * blckDim + tid;
```

Comment:
Why do you pass `blckDim` as a variable, why not use the builtin `blockDim.x`?

Reply:
Just thought it would look slightly cleaner in SYCL to call `item.get_local_range(0)` only once and store it as a variable instead of calling it everywhere, and equivalently just "copied" this into CUDA